interface/orders/types.php

```diff
@@ -239,7 +239,7 @@ function refreshFamily(id, haskids) {
 
     // edit/add a node
     function handleNode(id, type, add, lab) {
-        var editTitle = '<i class="fa fa-pencil" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?> + ' ';
+        var editTitle = '<i class="fa fa-pencil-alt" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?> + ' ';
```

Comment:
We should not be setting width in pixels here and should be using the FontAwesome width classes

Reply:
Hello @tywrenn 

Should I have to replace inline-style width to FontAwesome class?